docs/network-flow-visibility.md

```diff
@@ -264,7 +264,7 @@ then please use the address:
 `<Ipfix-Collector Cluster IP>:<port>:<TCP|UDP>`
 * If you have deployed the [ELK
 flow collector](#deployment-steps-1), then please use the address:  
-`<Logstash Cluster IP>:4739:UDP`
+`<Logstash Cluster IP>:4739:UDP` or `<Logstash Cluster IP>:4738:TCP`
```

Comment:
why a different port for TCP?

Reply:
I tired using same port for TCP and UDP and found there will be decoding issues in logstash, which may need further investigation. So I separated it to two different ports.

Reply:
Cleaned my local setting and had another try. It is working now with one port.